devicetypes/smartthings/zooz-4-in-1-sensor.src/zooz-4-in-1-sensor.groovy

```diff
@@ -23,8 +23,8 @@ metadata {
 		capability "Health Check"
 		capability "Tamper Alert"
 
-		fingerprint mfr: "027A", prod: "2021", model: "2101", deviceJoinName: "Zooz 4-in-1 sensor"
-                fingerprint mfr: "0109", prod: "2021", model: "2101", deviceJoinName: "ZP3111US 4-in-1 Motion"
+		fingerprint mfr: "027A", prod: "2021", model: "2101", deviceJoinName: "Zooz Multipurpose Sensor" //Zooz 4-in-1 sensor
+		fingerprint mfr: "0109", prod: "2021", model: "2101", deviceJoinName: "ZP3111US Multipurpose Sensor" //ZP3111US 4-in-1 Motion
```

Comment:
Isn't this supposed to be Zooz?

Reply:
No. It was just added to this DTH (devws) because it fit

Reply:
I just said that because I found ZP3111US mentioned on a Zooz user manual but I see the original PR now. Shouldn't this be "Vision" then?